conf/test.config

```diff
@@ -20,23 +20,23 @@ params {
   pyprophet_pi0_start = 0
   pyprophet_pi0_end = 0
   pyprophet_pi0_steps = 0
+  pyprophet_fdr_ms_level = 'ms1ms2'
```

Comment:
with test files there is always the danger that people take parameters for their normal analysis. Maybe document if non-standard parameter are used e.g., to make the tests run

Reply:
ok, will do